tools/extract_metadata.py

```diff
@@ -30,10 +30,12 @@ def is_wrapper_function(module, function):
       return False
     if opcode == OpCode.CALL:
       callee = module.read_uleb()  # noqa
-    elif opcode == OpCode.END:
-      break
+    elif opcode in (OpCode.LOCAL_GET, OpCode.LOCAL_SET, OpCode.END):
+      pass
```

Comment:
This needs to read the LEB of the local index, doesn't it? Otherwise it'll parse the index as the next instruction and likely get confused.
```suggestion
      index = module.read_uleb() # noqa
```

Reply:
Oh yeah.. i wonder how that wasn't causing failures.  Will double check.

Reply:
Fixed.